python/ray/experimental/sgd/pytorch/distributed_pytorch_runner.py

```diff
@@ -88,11 +88,15 @@ def step(self):
 
     def get_state(self):
         """Returns the state of the runner."""
+        cpu_state_dicts = []
+        for model in self.models:
+            state_dict = model.module.state_dict()
+            for k, v in state_dict.items():
+                state_dict[k] = v.cpu()
+            cpu_state_dicts += [state_dict]
         return {
             "epoch": self.epoch,
-            "models": [
-                model.module.cpu().state_dict() for model in self.models
```

Comment:
Why is just moving the entire model to cpu not sufficient? can you add a comment?

Reply:
fixed